models/User.js

```diff
@@ -18,7 +18,8 @@ User.add({
     initial: true,
     bucket: bucket,
     destination: 'avatars',
-    publicRead: true
+    publicRead: true,
+    autoCleanup: true,
```

Comment:
If the name is the same, do we really need to delete and write the same data?

Reply:
No, this commit has been removed.

Updated.